crates/primitives/starknet/src/crypto/commitment/mod.rs

```diff
@@ -57,7 +56,10 @@ impl<T: CryptoHasherT> CommitmentTree<T> {
 /// # Returns
 ///
 /// The transaction commitment, the event commitment and the event count.
-pub fn calculate_commitments<T: CryptoHasherT>(transactions: &[Transaction], events: &[EventWrapper]) -> (H256, H256) {
+pub fn calculate_commitments<T: CryptoHasherT>(
```

Comment:
No need to touch this file on this PR because of #667

Reply:
haha nice, you already fixed that.